docker-app/qfieldcloud/core/templates/admin/project_files_widget.html

```diff
@@ -169,6 +169,17 @@
                     .then((files) => refreshTableContents({ files }))
                     .catch((error) => refreshTableContents({ error }));
             };
+            const toFixed1 = (n, f) => (n/f).toFixed(1);
```

Comment:
Avoid this kind of super functional type of writing, you saved 2-3 characters in the original place of calling.

```
toFixed1(n, 10 ** 3) + " KB";
```
vs
```
((n / 10) ** 3).toFixed(3)+ " KB";
```

I would propose we assume the relevant digits are 3, there are quite a lot of files in GIS that can fit within 100 bytes, which will show as 0 KB in your case.

Finally, any chance that we add this in a `<span title="{n} bytes">{N} KB</span>`, so the supporter has the full information about the file size?

Reply:
Oh it's not really a matter of style, it's a matter of having 1 thing to change instead of 3 as soon as we need refactoring. And I am not counting call sites where it might be needed in the future.

Yes to the other two things.

Reply:
I think I get your intent.

But I believe it is more or less matter of style this time with this contents, because trying to abstract this will take more maintanance and congnitive effort to understand in the future.

I think you can achieve pretty much the same without the need of an extra function, just by refactoring your current  `humanReadableSizes`. Use single return in the end, something like `return `<span title="${bytes} bytes">${sizeValue} ${sizeUnit}</span>` and calculate `sizeValue`, and `sizeUnit` in the `if`s/`switch` statements before.

BTW `humanReadableSizes` is probably not the best name, especially the plural in the name, something like `bytesToHumanReadableSize` or `filesize10` as in Django will be better.

Reply:
Can you still please remove the now renamed `toFixed3` function?